terraform/eks-cluster/commands/run.sh

```diff
@@ -22,8 +22,8 @@ bazel build server \
   --remote_instance_name=$(date +%s) \
   --host_cpu=k8 --cpu=k8 \
   --crosstool_top=@buildbuddy_toolchain//:ubuntu1604_cc_toolchain_suite \
-  --host_platform=@buildbuddy_toolchain//:platform_linux \
-  --platforms=@buildbuddy_toolchain//:platform_linux \
+  --host_platform=//platforms:buildbuddy_rbe_linux_x86_64_default \
+  --platforms=//platforms:buildbuddy_rbe_linux_x86_64_default \
```

Comment:
Not only we don't need to update this, but we probably need to rework a lot of these flags for it to work correctly now that we are no longer using `cc_toolchain_suite` 🤔 

I would prefer to leave this out and address this in a separate PR.

Reply:
👍